dist/main.js

```diff
@@ -0,0 +1 @@
+index.js
```

Comment:
How does this work exactly? Is this executed as a bash script or is this some kind of magic javascript or actions syntax?

Reply:
This is how GitHub shows symlinks. 

Both `main.js` and `post.js` are symlinked to `index.js`. 

So when you run `node main.js` it will technically evaluate the `index.js`, but it can read with what arguments it was evaluated with and act accordingly. 

Github action runs `node $FULL_PATH/main.js`, so the `index.js` will take the first argument and see what file was executed and run it.

Reply:
Addressed by a comment.